corehq/apps/app_manager/static/app_manager/js/supported_languages.js

```diff
@@ -1,4 +1,11 @@
-hqDefine('app_manager/js/supported_languages', function () {
+hqDefine('app_manager/js/supported_languages',[
+    "jquery",
+    "knockout",
+    "underscore",
+    "hqwebapp/js/initial_page_data",
+    "hqwebapp/js/main",
+    "hqwebapp/js/knockout_bindings.ko",
```

Comment:
Could you comment what the binding is that's used here?

Reply:
Hey, I should have been clearer, I meant adding a comment directly in the code :)  `knockout_bindings.ko` is basically a big junk drawer of custom bindings. I'd like to be able to reorganize it at some point, and that'll be easier if we have info about which specific bindings different modules depend on.

Reply:
ha ha..This turned out to be funny.. ya sure I add the comment there.